plugins/kubernetes/ingress_kong.go

```
@@ -229,6 +229,9 @@ func (x *Kubernetes) createKongIngress(e transistor.Event) error {
 
 		if len(route.Methods) > 0 {
 			methods := strings.Split(strings.ToUpper(strings.Join(route.Methods, ",")), ",")
```

Could also do a `strings.Replace(..., " ", "", -1)`, but I'm fine with approving this for now

---

Why would `strings.Replace()` be preferred in this situation?

---

We may also want to return an error if we find spaces in indexes that are not at the front or end of the string e.g. `route1, route    2, route3`

---

I think that's unnecessary in this case. Kong will respond with a failure directly. This is mostly for user convienience. If the user is attempting to create a method of `POST ABC` then it will be validated by the kong api.

This is mostly to help users who add spaces when making their list `POST, GET`, and it's not totally clear when the API responds with validation that it's literally attempting to add ` GET`